src/pages/admin/company-profile/components/company-profile-social-media-tab/CompanyProfileSocialMediaTab.module.scss

```
@@ -72,3 +73,68 @@
+.social-media-tab-add :global(.option-selected),
+.social-media-tab-add :global(.option-selected:hover) {
+    background-color: c.$dropdown-hover;
+}
+
+.social-media-tab-add :global(.option-value),
+.social-media-tab-add :global(.option-value-placeholder) {
+    color: inherit;
+}
+
+.social-media-tab-add :global(.placeholder-arrow .icon-img) {
+    fill: currentColor;
```

Stylelint (17.5.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)